tests/foreman/rhai/test_rhai.py

```diff
@@ -130,6 +131,9 @@ def test_unregister_system_from_rhai(self):
                 session.nav.click(
                     locators['insights.unregister_button']
                 )
+                self.browser.refresh()
+                time.sleep(30)
```

Comment:
@oshtaier I completely understand your concern and thought of the condition thing as well, but then that would be like 'uploading data until we get a status code:412(which is expected)'. I have written my concern to the team but haven't got any updates, so as soon as I get something better for this test to deal with, I shall update it. Till then we could merge it(if you think it is ok to do so) as the test has been failing on Jenkins for a past few builds.